client/query.go

```diff
@@ -154,3 +155,48 @@ func Blocks(ctx context.CLIContext, startingHeight *big.Int) ([]store.Block, err
 
 	return blocks, nil
 }
+
+// Returns confirm sig results for given position
+// Trusts connected full node
+func Signatures(ctx context.CLIContext, position plasma.Position) ([]byte, error) {
```

Comment:
Can we rename this to `ConfirmSignatures`. Don't want to confuse this with normal transaction signatures